modules/cluster-logging-log-forwarding-about.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-log-forwarding.adoc
+
+[id="cluster-logging-log-forwarding-about_{context}"]
+= About cluster log forwarding
+
+The {product-title} cluster log forwarding feature enables administrators to send collected log data 
+to external systems outside the {product-title} cluster. 
+
+[NOTE]
+====
+Enabling the log forwarding feature disables the xref:../../logging/config/cluster-logging-external.adoc#cluster-logging-external[default log forwarding feature].
+====
+
+Log forwarding enables you to use your existing logging service with your {product-title} cluster. You can send each type of log to different systems
+allowing you to control who in your organization can access each type. Optional TLS support ensures that you can send logs using secure communication
+as required by your organization.
+
+Log forwarding uses a combination of _outputs_ and _pipelines_ defined in the Log Forwarding Custom Resource. An output is the destination for log data and a pipeline defines simple routing for one source to one or more outputs. 
+
+An output can be either:
+
+* `elasticsearch` to forward logs to an external Elasticsearch cluster, specified by server name or FQDN.
```

Comment:
@sichvoge do we need to provide reference to the format of the messages and to what index and document type here since we don't allow configuration of it?

Reply:
@sichvoge Any thoughts ^^